### DISCUSSION TITLE
PDF/A-1b Standard ISO

### DISCUSSION THREAD

**Discussion by dado70** [pdfarranger/pdfarranger#691] [Q&A] [answered]
Hi everybody,
Is pdfarranger compatible with PDF/A-1b and other Standard ISO PDF?

dado70

**Comment by dreua** [comment, marked as answer]
What exactly do you mean by "compatible"? I'm pretty sure it can read PDFs of this standard, but the exported PDFs will probably not be PDF/A conformant. We use pikepdf for PDF-Manipulation which claims to support [PDF versions 1.1 to 1.7](https://github.com/pikepdf/pikepdf#features). The thumbnails on the other hand are rendered by poppler + cairo, not sure if that is important to your "compatibility" as well.

Maybe do some tests with PDFs and validators and check for yourself?

**Reply by dado70** [reply]
Hi thanks for replay, 
for italian law sometimes I must to create PDF/A-1b. I do that from Libreoffice o PDFCreator (pdfforge.org).
If I do that pdf from pdfarranger the files created seems not to be PDF/A-1b.

see: 
https://en.wikipedia.org/wiki/PDF/A
https://urbanistica.comune.modena.it/info/PDFA/PDF-A/index_PDF-A.html

**Reply by dreua** [reply]
I guess you have your answer then? I'm sorry we don't have better news for you.

**Reply by dado70** [reply]
Can I hope for an implementation? :dancing_men:

**Reply by dreua** [reply]
I don't think that will happen, sorry.

**Reply by jeromerobert** [reply]
Just to confirm the dreua answer: https://github.com/qpdf/qpdf/issues/85

**Reply by jeromerobert** [reply]
@dado70 Concidering the PDF/A wikipedia page, here is what I think to be our status:
* **Language specification**: I guess that PDF-Arranger do not remove it. I don't know what happen if we have multiple languages
* **Hierarchical document structure**: PDF-Arranger remove that. Merging document structures would be very not trivial.
* **Tagged text spans and descriptive text for images and symbols**: I don't expect PDF-Arranger to remove that so it should be fine
* **Character mappings to Unicode**: I don't expect PDF-Arranger to remove that so it should be fine
